tests/integration/targets/openssh_keypair/tests/validate.yml

```diff
@@ -138,3 +138,34 @@
   assert:
     that:
       - privatekey8_result_force is changed
+
+- name: Check that password protected key with passphrase was regenerated
+  assert:
+    that:
+      - privatekey9_modified_result is changed
+  when: cryptography_version.stdout is version('3.0', '>=')
```

Comment:
I would probably put all these asserts with the same `when:` into `block:` so you don't have to repeat the `when:` everywhere.